WordPress/src/main/java/org/wordpress/android/ui/sitecreation/usecases/FetchVerticalsHeaderInfoUseCase.kt

```diff
@@ -0,0 +1,62 @@
+package org.wordpress.android.ui.sitecreation.usecases
+
+import org.greenrobot.eventbus.Subscribe
+import org.greenrobot.eventbus.ThreadMode
+import org.wordpress.android.fluxc.Dispatcher
+import org.wordpress.android.fluxc.store.Store
+import org.wordpress.android.fluxc.store.VerticalStore
+import org.wordpress.android.fluxc.store.VerticalStore.VerticalErrorType
+import javax.inject.Inject
+import kotlin.coroutines.experimental.Continuation
+import kotlin.coroutines.experimental.suspendCoroutine
+
+/**
+ * Transforms EventBus event to a coroutines.
```

Comment:
We might want to add the even this use case handles.

Also, `a coroutine`. 😉

Reply:
👍